package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "osl-website",
+  "version": "1.0.0",
+  "description": "Migrated from our static [Pelican site](https://github.com/osuosl/osuosl-pelican)",
+  "main": "",
+  "scripts": {
+    "serve": "pagefind --site public --serve",
+    "format": "prettier . -w",
+    "lint": "markdownlint-cli2 ./content/**/*.md"
```

Comment:
A few things to note here:
- Can add documentation on how to use these tools locally and installing them?
- I'd like to get some of these incorporated with github actions:
  - https://github.com/marketplace/actions/prettier-action
  - https://github.com/DavidAnson/markdownlint-cli2-action
  - https://docs.github.com/en/code-security/dependabot/ecosystems-supported-by-dependabot/supported-ecosystems-and-repositories
    - This can be used for automatically bumping this file and the other github actions. I may need to make a few tweaks in the repo to get it to work. We have this in a few other projects (python however, look at the docs repo)

Reply:
Documentation has been added to the README.

Reply:
GitHub actions noted below, for Dependabot I checked the Security Tab and it says I do not have permission to enable it.

![image](https://github.com/user-attachments/assets/6421143f-1859-43b5-92be-1c5d03e14991)

Reply:
Leaving this unresolved until I know what to do for the dependabot thing, feel free to resolve it yourself if you feel satisfied with the notes above.

Reply:
@FireIsGood just enabled dependabot.